web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -1840,8 +1839,8 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
     val isForm = !attrs.get("form").toList.isEmpty
 
     val eagerEval: Boolean =
-      (attrs.get("eager_eval").map(toBoolean) or
-        findNSAttr(attrs, "lift", "eager_eval").map(toBoolean) or
+      (attrs.get("eager_eval").map(toBoolean) orElse
```

Comment:
If they will import both Helpers._ and Box._, the have to. Other possibility is to explicitly convert first element of chain and other will be implicitly converted without Box._ import. eg.
```scala
ttrs.get("eager_eval").map(toBoolean).toBox or  findNSAttr(attrs, "lift", "eager_eval").map(toBoolean)
```